kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flight_finder/FlightDatabase.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.good.patterns.flight_finder;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public final class FlightDatabase
+{
+    private final Set<Flight> flights;
+
+    public FlightDatabase()
+    {
+        this.flights = new HashSet<>();
+        this.flights.add(new PrivateFlight("Warszawa", "Gdańsk"));
+        this.flights.add(new PrivateFlight("Warszawa", "Poznań"));
+        this.flights.add(new PrivateFlight("Warszawa", "Katowice"));
+        this.flights.add(new PrivateFlight("Warszawa", "Wrocław"));
+        this.flights.add(new PrivateFlight("Poznań", "Kraków"));
+        this.flights.add(new PrivateFlight("Poznań", "Gdańsk"));
+        this.flights.add(new PrivateFlight("Poznań", "Warszawa"));
+        this.flights.add(new PrivateFlight("Wrocław", "Gdańsk"));
+        this.flights.add(new PrivateFlight("Gdańsk", "Warszawa"));
+        this.flights.add(new PrivateFlight("Gdańsk", "Kraków"));
+        this.flights.add(new PrivateFlight("Katowice", "Warszawa"));
+        this.flights.add(new PrivateFlight("Kraków", "Gdańsk"));
+        this.flights.add(new PrivateFlight("Kraków", "Poznań"));
+        this.flights.add(new PrivateFlight("Wrocław", "Warszawa"));
```

Comment:
this. tutaj jest zbędne, aczkolwiek jeśli dla Ciebie kod jest w takiej postaci łatwiej czytelny, to zostaw to tak.

Reply:
Zgodnie z rozmową deklaracja zamierzona.